### PR TITLE
Deprecated group_ids attribute

### PR DESCRIPTION
**Pull request by vmanilo**
Resolves https://github.com/Twingate/terraform-provider-twingate/issues/242

## Changes
- deprecated attribute `group_ids`

### CODE REVIEW

**Review by ekampf — APPROVED** [review]

File: twingate/internal/provider/resource/resource.go [hunk 1]
```diff
@@ -66,6 +66,7 @@ func Resource() *schema.Resource { //nolint:funlen
 				Elem:        &schema.Schema{Type: schema.TypeString},
 				Optional:    true,
 				Description: "List of Group IDs that have permission to access the Resource, cannot be generated by Terraform and must be retrieved from the Twingate Admin Console or API",
+				Deprecated:  "Configure `access` block instead. This attribute will be removed in the next version of the provider.",
```
Comment (alexmensch) [inline twingate/internal/provider/resource/resource.go:69]: Let's change this to: "The group_ids argument is now deprecated, and the new access block argument should be used instead. The group_ids argument will be removed in a future version of the provider."

**Review by alexmensch — CHANGES_REQUESTED** [review]

File: twingate/internal/provider/resource/resource.go (hunk 1 above)
Comment (vmanilo) [reply twingate/internal/provider/resource/resource.go]: updated

**Review by alexmensch — APPROVED** [review]

**Comment by tjstansell** [comment]
Hm... odd that you'd merge this before an `access {}` block even exists.  I'm used to seeing new features added and tested for a few releases and _then_ the old mechanism deprecated. :)

**Comment by rebmcr** [comment]
> Hm... odd that you'd merge this before an `access {}` block even exists. I'm used to seeing new features added and tested for a few releases and _then_ the old mechanism deprecated. :)

Indeed, looks like the `access{}` block might fulfill a feature request we made, so hopefully this is just out-of-sequence comms!

**Comment by alexmensch** [comment]
This PR just marks the argument as deprecated, we won't actually remove support until later on.

The next release will include this plus PR #249 so they will be released together. Merge != release!

**Pull request merged**